component_sdk/python/kfp_component/google/storage/_common_ops.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+
+def is_gcs_path(path):
```

Comment:
Since this function is in a storage directory, maybe we should add verification.
There are name requirements for GCS path: https://cloud.google.com/storage/docs/naming
For example, the following two requirements are important and easy to check:
1) Bucket names must start and end with a number or letter.
2) Bucket names must contain 3 to 63 characters.

Reply:
I purposely skip those checks in a lot of places because we are passing user provided data to GCP APIs and the APIs will validate the it and report bad request error to the user and we will show it in the log. The validation will be a dup work from the server code did and we have to maintain it if any rules are changed in the future. The check I have it here is for a logic in my code which differs from gcp path and local path.